pkg/client/uri.go

```diff
@@ -42,12 +44,26 @@ func (as *UriService) Create(ctx context.Context, input CreateUriInput) (*Create
 func (as *UriService) Read(ctx context.Context, id string) (*ReadUriResult, error) {
 	log.Printf("read uri request. id=%s", id)
 
-	resp, err := getUriById(ctx, as.client.gql, id)
+	operation := func() (getUriByIdEntitiesEntityQueriesByIdEntity, error) {
+		resp, err := getUriById(ctx, as.client.gql, id)
+
+		if err != nil {
+			return nil, err
+		}
+
+		if resp.Entities.ById == nil {
+			return nil, backoff.RetryAfter(1)
+		}
+
+		return *resp.Entities.ById, nil
+	}
+
+	uriPtr, err := backoff.Retry(ctx, operation, backoff.WithBackOff(backoff.NewExponentialBackOff()), backoff.WithMaxElapsedTime(5))
```

Comment:
One more question.

Is it possible that `backoff.Retry` could return nil, nil?